packages/react-components/react-dialog/library/src/components/DialogBackdropMotion.ts

```diff
@@ -1,16 +1,9 @@
 import { createPresenceComponent, motionTokens } from '@fluentui/react-motion';
+import { createFadePresence } from '@fluentui/react-motion-components-preview';
```

Comment:
```suggestion
import { FadeRelaxed } from '@fluentui/react-motion-components-preview';
```

Reply:
Also add `@fluentui/react-motion-components-preview` as a dependency in `package.json`.

Reply:
I was added